app/config/security.yml

```diff
@@ -73,14 +73,12 @@ nelmio_security:
                 - 'none'
             style-src:
                 - 'self'
-                - 'netdna.bootstrapcdn.com'
                 - 'unsafe-inline'
             script-src:
                 - 'self'
-                - 'ajax.googleapis.com'
+                - 'unsafe-inline'
```

Comment:
This change goes against the concept of defence in depth because this will effectively disable the XSS protection mechanism of CSP. Could the googleapis and bootstrapcdn not be moved to the dev config. That way it'll be secure until the webserver removes the csp headers.

Reply:
I think that one can be removed entirely, might have missed that while creating the exception in the Apache config.